src/ray/core_worker/experimental_mutable_object_manager.cc

```diff
@@ -428,6 +442,18 @@ Status MutableObjectManager::SetErrorAll() {
   return ret;
 }
 
+std::unique_ptr<std::chrono::steady_clock::time_point>
+MutableObjectManager::ToTimeoutPoint(int64_t timeout_ms) {
+  if (timeout_ms == -1) {
+    return nullptr;
+  }
+  auto now = std::chrono::steady_clock::now();
```

Comment:
Please use the type instead of `auto`

Reply:
Can you elaborate a bit why is that preferred?
I think it is actually more readable to use `auto`. This code is very local and compiler would help us catch any type violations.